src/pages/Transaction/TransactionCellScript/CellInfoDataView.tsx

```diff
@@ -0,0 +1,74 @@
+import { useMemo, useRef, useState } from 'react'
+import { Select } from 'antd'
+import { useTranslation } from 'react-i18next'
+import { Root, Trigger, Portal, Content } from 'selection-popover'
+import styles from './styles.module.scss'
+
+const DECODER = {
+  utf8: 'UTF-8',
+  number: 'Number',
+  // todo: support decode address & Big-Endian
```

Comment:
```suggestion
  // todo: support decode address & Big-Endian & JSON string & Address
```

Reply:
Here I have a bit of a question, what should be the decode rule for address? Is hex used as a public key and then parse to address or what does it mean?

And are the rules for decoding json different from utf-8?

Reply:
I think parsing address into a script is pretty enough